<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#closeButton, #miniButton{
	background-color: rgba(255,255,255,30);
	border: none;
	border-radius: 5px;
	border-bottom: 1px solid rgba(51, 51, 102, 255);
	border-right: 1px solid rgba(51, 51, 102, 255);
	color: rgba(51, 51, 102, 255);
}

#closeButton:hover, #miniButton:hover{
	background-color: rgba(51, 51, 102, 50);
}

#closeButton:pressed, #miniButton:pressed{
	background-color: rgba(51, 51, 102, 100);
	border-top: 2px solid rgba(51, 51, 102, 255);
	border-left: 2px solid rgba(51, 51, 102, 255);
	padding-top: 2px;
	padding-left: 1px;
}</string>
   </property>
   <widget class="QWidget" name="widgetest" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lpButton, #bookButton, #gameButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0, stop:0 rgba(34,35,46,255), stop:1 rgba(85, 98, 112, 255));
}

#lpButton:hover, #bookButton:hover, #gameButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0, stop:0 rgba(40, 67, 98, 255), stop:1 rgba(105, 118, 132, 255));
}

#lpButton:pressed, #bookButton:pressed, #gameButton:pressed{
	padding-top: 5px;
	background-color: rgba(105, 118, 132, 255);
}</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>281</width>
       <height>711</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(51, 153, 204, 255), stop:1 rgba(51, 51, 102, 255));
	border-radius: 5px;
}</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Lorum Ipsum BOIIII</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>121</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../test.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="text">
       <string>USER #1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="lpButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(255,255,255,100);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string>LP's</string>
      </property>
     </widget>
     <widget class="QPushButton" name="miniButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bookButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(255,255,255,100);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string>BOOKS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gameButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>550</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: none;
border-bottom: 2px solid rgba(255,255,255,100);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string>GAMES</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="search">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Search...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>look</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>651</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget2{
	background-color: rgba(34,35,46,255);
	border-radius: 10px;
}
</string>
    </property>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Toevoegen</string>
     </property>
    </widget>
    <widget class="QLabel" name="latest_additions">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>LATEST ADDITIONS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="account">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>26</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Verwijder</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>601</width>
       <height>481</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>599</width>
        <height>479</height>
       </rect>
      </property>
      <widget class="QLabel" name="coverLabel_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="titleLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name of thing</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>widget2</zorder>
   <zorder>widgetest</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
